src/app/tests/suites/TestGroupKeyManagementCluster.yaml

```diff
@@ -41,117 +40,62 @@ tests:
               - name: "groupId"
                 value: 11
 
-    - label: "Add Group 2"
-      disabled: true
-      cluster: "Groups"
-      endpoint: 1
-      command: "AddGroup"
+    - label: "KeySet Write"
+      command: "KeySetWrite"
       arguments:
           values:
-              - name: "groupId"
-                value: 12
-              - name: "groupName"
-                value: "Group #2"
-      response:
-          values:
-              - name: "status"
-                value: 0
-              - name: "groupId"
-                value: 12
+              - name: "groupKeySet"
+                value:
+                    {
+                        groupKeySetID: 101,
+                        securityPolicy: 0,
+                        epochKey0: "\xa0\xa1\xa2\xa3\xa4\xa5\xa6\xa7\xa8\xa9\xaa\xab\xac\xad\xae\xaf",
+                        epochStartTime0: 1110000,
+                        epochKey1: "\xa0\xa1\xa2\xa3\xa4\xa5\xa6\xa7\xa8\xa9\xaa\xab\xac\xad\xae\xaf",
```

Comment:
Shouldn't the values here be `\xb0\xb1.....`, so ephochKey1 is not the same as epochKey2?

Reply:
Yes it should. Fixed.